analyzers/Gun.py

```diff
@@ -1,6 +1,8 @@
 from heppy.framework.analyzer import Analyzer
 from heppy.papas.pdt import particle_data
-from heppy.particles.tlv.particle import Particle 
+from heppy.papas.pfobjects import Particle #so that Gun can be used in papas (needs uniqueid)
```

Comment:
Made Gun have an option to produce PapasParticle or tlv particle. Hope this is better?